Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -322,7 +354,9 @@ class BrowserViewController: UIViewController {
         urlBar.translatesAutoresizingMaskIntoConstraints = false
         urlBar.delegate = self
         urlBar.tabToolbarDelegate = self
-        header = BlurWrapper(view: urlBar)
+        header = BlurWrapper(view: urlBarTopTabsContainer)
```

Comment:
You might run into issues down the road when implementing the views for the tabs if everything is inside this blur wrapper. Not sure though. I'm okay with starting with this and fixing later if thats the case.